frontend/src/Components/NewRunConfig/AdvancedSettings/AdvancedSettings.tsx

```diff
@@ -0,0 +1,111 @@
+import { Button, Card, Elevation, FormGroup, NumericInput } from "@blueprintjs/core";
+import { SubmitSimulation } from "../NewRunState";
+import { SimConfig } from "../SimConfig";
+import "./AdvancedSettings.css";
+
+export default function AdvancedSettingsMenu(state: any) {
+
+  const config = state[0]
+  const setConfig = state[1]
+
+  function configHandler<Key extends keyof SimConfig>(value: number, keyString: any){
+    var key:Key = keyString // converting keyString to type Key
+    config[key]=value;
+    setConfig(config)
+  }
+
+  function SubmitState(){
+    const configJSON = JSON.stringify(state[0]);
+    SubmitSimulation(configJSON)
+  }
+
+  return(
+    <div
+    className="modal custom fade"
+    id="testModal"
+    data-backdrop="false"
+    tabIndex={-1}
+    aria-labelledby="staticBackdropLabel"
+    aria-hidden="true"
+    >
+      <div className="modal-dialog">
+      <div className="modal-content">
+        <Card interactive={true} elevation={Elevation.TWO}>
+        <FormGroup helperText="In Days..." label="Simulation Length" labelFor="text-input" >
```

Comment:
These can be changed to a for loop and importing the fields from a config file to make it cleaner